lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -127,6 +130,26 @@ def define_protocol(self):
         else:
             self.params['protocol'] = 'http'
 
+    def sanitize_keys(self, data):
+        if isinstance(data, dict):
+            items = {}
+            for k, v in data.items():
+                try:
```

Comment:
Why not something like:
```.py
  new_key = self.key_map.get(k)
  if new_key is None:
    new_key = re.sub('([a-z0-9])([A-Z])', r'\1_\2', k).lower()
  items[new_key] = self.sanitize_keys(v)
```
(instead of the whole `try/except` block)

Reply:
I haven't made this change yet, but I will.